src/environment.rs

```diff
@@ -0,0 +1,75 @@
+use eel::Network;
+
+#[derive(Clone, Copy, Debug)]
+pub enum EnvironmentCode {
+    Local,
+    Dev,
+    Stage,
+    Prod,
+}
+
+pub(crate) struct Environment {
+    pub network: Network,
+    pub backend_url: String,
+    pub backend_health_url: String,
+    pub lsp_url: String,
+    pub lsp_token: String,
+    pub esplora_url: String,
+    pub rgs_url: String,
+}
+
+impl Environment {
+    pub fn for_(environment: EnvironmentCode) -> Environment {
```

Comment:
```suggestion
    pub fn for_(environment: EnvironmentCode) -> Self {
```

I get the point with the function name `for_`, but it is a very uncommon way of naming functions. How about `load(environment: EnvironmentCode)` or something?